src/rust/engine/process_execution/src/lib.rs

```diff
@@ -72,27 +72,30 @@ use fs::RelativePath;
 
 #[derive(PartialOrd, Ord, Clone, Copy, Debug, Eq, PartialEq, Hash, Serialize, Deserialize)]
 pub enum Platform {
-  Darwin,
-  Linux,
+  MacOsX86_64,
+  MacOsArm64,
+  LinuxX86_64,
```

Comment:
Maybe this would be less awkward? Technically you're supposed to do UpperCamelCase for enum variants, but we can turn off Clippy (Rust linter) for this one. 

```rust
  Macos_x86_64,
  Macos_arm64,
  Linux_x86_64,
```

Reply:
@stuhood thoughts?

Reply:
Yea, allowing snake case here probably makes sense. Can skip the lint with:
```
#[allow(non_snake_case)]
enum .. {
  ..
}
```